scipy/optimize/_lsq/least_squares.py

```diff
@@ -446,7 +441,7 @@ def least_squares(
             * -1 : a lower bound is active.
             *  1 : an upper bound is active.
 
-        Might be somewhat arbitrary for 'trf' method as it does strictly
+        Might be somewhat arbitrary for 'trf' method as it only uses strictly
```

Comment:
I searched "Numerical Optimization" book for the word "iterates", it is used mostly as or "generates a sequence of iterates"

So I suggest "as it generates a sequence of strictly feasible iterates" ("uses" is also good, "does" is not correct).